include/grpcpp/support/proto_buffer_reader.h

```diff
@@ -120,6 +122,49 @@ class ProtoBufferReader : public grpc::protobuf::io::ZeroCopyInputStream {
   /// Returns the total number of bytes read since this object was created.
   int64_t ByteCount() const override { return byte_count_ - backup_count_; }
 
+#ifdef GRPC_CORD_SUPPORT_ENABLED
+  /// Read the next `count` bytes and append it to the given Cord.
+  bool ReadCord(absl::Cord* cord, int count) override {
```

Comment:
Should this be marked experimental first before it goes through a design review?

Reply:
Also, does it need a gRFC?

Reply:
Also, tests maybe?

Reply:
This doesn't need design review or gRFC as this is a gRPC-specific implementation of Protobuf's interface. Maybe we can add a test.

Reply:
Unit tests are added